src/main/java/org/llorllale/cactoos/matchers/MatchesBefore.java

```diff
@@ -49,22 +53,60 @@
  * @param <T> Type of the scalar's value
  * @since 1.0.0
  */
+@SuppressWarnings({
+    "PMD.AvoidCatchingGenericException",
+    "PMD.CallSuperInConstructor",
+    "PMD.ConstructorOnlyInitializesOrCallOtherConstructors"
+    })
 public final class MatchesBefore<T> extends MatcherEnvelope<T> {
+    /**
+     * Time unit.
+     */
+    private static final String UNIT = "milliseconds";
```

Comment:
@andreoss can you renamed `UNIT`to `TIME_UNIT`? I think this name more meaningful.

Reply:
@fabriciofx Done